### PR TITLE
Add Evaluate action to Check Enforcer check-run.

### PR DESCRIPTION
**Pull request by mitchdenny**
This PR adds an Evaluate action button to Check Enforcer. At this point it adds the button but doesn't do anything. This is just to get some telemetry that shows how the payload changes when the action is received so that I can add it to Octokit and add appropriate tests.

### CODE REVIEW

File: tools/check-enforcer/Azure.Sdk.Tools.CheckEnforcer/Handlers/CheckRunHandler.cs [hunk 1]
```diff
@@ -14,31 +14,6 @@ namespace Azure.Sdk.Tools.CheckEnforcer.Handlers
 {
     public class CheckRunHandler : Handler<CheckRunEventPayload>
     {
```
Comment (JimSuplizio) [inline tools/check-enforcer/Azure.Sdk.Tools.CheckEnforcer/Handlers/CheckRunHandler.cs:16]: @mitchdenny, what are ramifications of removing these EventIds that the logger was using?

**Comment by mitchdenny** [comment]
Thanks. The event ids were added because I thought that they would be useful but turns out they aren't in practice because I just use various joins and parse with operations to get at what I want in the telemetry. I'll be removing the rest soon.

File: tools/check-enforcer/Azure.Sdk.Tools.CheckEnforcer/Handlers/CheckRunHandler.cs (hunk 1 above)
Comment (mitchdenny) [reply tools/check-enforcer/Azure.Sdk.Tools.CheckEnforcer/Handlers/CheckRunHandler.cs:16]: Nothing in practice. It was just an idea that didn't pan out.

**Pull request merged**